Sources/TSCBasic/Process.swift

```diff
@@ -370,6 +404,17 @@ public final class Process: ObjectIdentifierProtocol {
         posix_spawn_file_actions_init(&fileActions)
         defer { posix_spawn_file_actions_destroy(&fileActions) }
 
+      #if os(macOS)
```

Comment:
Glibc 2.29+ has `posix_spawn_file_actions_addchdir_np` as well (same signature), so this can work on Linux if you want to do that. I'm not sure which distros have which versions (2.29 was released Feb 2019) so it might need an `#if __GLIBC_PREREQ(2, 29)` depending on what Linux flavors the SwiftUI supports.

Reply:
cool! I'll leave that as a potential improvement for someone if they wanted this support in Linux